patient-view-be/src/main/java/spvproject/patientviewbe/controller/PainAssessmentController.java

```diff
@@ -0,0 +1,23 @@
+package spvproject.patientviewbe.controller;
+
+
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import org.springframework.web.bind.annotation.RestController;
+
+import spvproject.patientviewbe.model.PainAssessmentModel;
+
+@RestController
+@RequestMapping("/painAssessment")
+public class PainAssessmentController {
+	
+	
+	@GetMapping("/first")
```

Comment:
Please rename the route so that it is something meaningful and intuitive. Routes should make sense what they do from their names. In this case I would suggest something like `/data`.
Your request will then be `GET /pain-assessment/data`